### PR TITLE
A short feature freeze while we decentralize RnD

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
A short feature freeze until phase 1/mapping of https://github.com/tgstation/tgstation/projects/12 is complete or two weeks from now when we give up, whichever is first.

Techwebs and circuits are both massive PRs and I'd like to free up maintainer time for them and spare the two PRs from competing balance changes/conflicts/etc (and encourage other people to help on them!)

As usual anything that was made before this freeze is exempt (as are fixes, but I'd prefer no big refactors that cause conflicts in the aforementioned PRs)

I will give a copy of Furi to whoever sprites the departmental lathes to a standard WJ approves

### CODE REVIEW

**Comment by ike709** [comment]
Instead of Furi can I have another game of equal value?

**Comment by Armhulen** [comment]
cool

**Comment by Qustinnus** [comment]
guess im sticking all my sounds in one PR after all kek

**Comment by Robustin** [comment]
So uh is there a post describing this approach? This is the biggest balance change in over a year, what will be the limiting aspect for each lathe? How will resource distribution be handled when there's no QM or the QM just resorts to giving his own department priority on minerals? What incentive does the RD have to research stuff his department can't access or use? 

I'm also going to make my standard complaint that you fought against this exact concept for engineering but here we are with "Techweb Points" but instead of coming up with new SM setups or advanced canister mixes you're screaming at the RD on command channel to invest Tech Boy Points into flight suits (which are easily more powerful than anything I tried to add to engineering).

**Comment by JJRcop** [comment]
With empty index
```SHELL
git commit --allow-empty -M "Short feature freeze: R&D Decentralization"
```

**Comment by davethwave** [comment]
It was just made so he could be filling it in as we speak or is going to take care of that part later. Also I think it might be made in such a way for the player base to make ideas and fill in the blanks instead of himself possibly?

**Comment by Robustin** [comment]
I mean this is Kor so of course he's not filling in any of the blanks.

**Comment by davethwave** [comment]
Also kor, make sure science gets one science lathe also, mainly focusing on bluespace, weird science, and stock parts.(and more random stuff probably)

**Comment by Rinoka** [comment]
Robustin, you raise good points, but I'd suggest making the way the lathes work be similar to normal lathes, and possibly add collection-bots that can carry small amounts of materials from QM to the other lathes (think mini-MULEs) to comp for the unique materials issue?

I think it is definitely an issue that this PR opens the door to potential complaints about RDs not picking the "optimal" choice, so something I'd prefer is that Techwebs have inherent randomization, akin to powers. The more people think of research as a random minigame, the less they're likely to hurl shit at an RD for not magically producing their favorite meta item in a round.

I think the major benefit of techwebbing, other than distributing Research and decentralizing robust power, is the ability to use randomness to reduce "HEY DO THIS FOR ME" demand. I'd prefer this form of research being genetics, rather than chems.

**Comment by davethwave** [comment]
instead of mini-mules how about using those integrated circuitry bots. Have a few spawn in at the beginning of the round with the right settings to pick up and deliver mats to bring more attention to the pr and its capability's in the game

**Comment by Rinoka** [comment]
Maybe something like that, plus the PR mentions adding custom-made bots - maybe add some upgraded/custom settings for bots to make integrated circuitry bots? Or a way to make a spawner for material delivery bots? It'd be interesting to see how that went, even if it had a high metal cost limiting how many could be made easily.

**Comment by Sarcalogo** [comment]
Furi is a good game outside of walking sections.

**Comment by KorPhaeron** [comment]
@Robustin redoing one GBP system we already have and are more or less stuck with is different than adding a new one

Most of the blanks are already filled in by the circuit/techweb PRs already

The incentives for the RD might be bad but we'll figure that out when we see how it is in practice

If the QM is being a dumb asshole/doesn't exist then security/HoP/captain/AI etc should deal with it the same as if any other job is missing.

Also you missed the forum discussion but I listed reviving your GBP thing as part of phase 2 of this

**Comment by KorPhaeron** [comment]
@ike709 no it is only furi

**Comment by lzimann** [comment]
Seems like a huge waste of time doing a feature freeze for this

**Comment by duncathan** [comment]
it's a good opportunity for people to finish their shit
pr tracker is ridiculous rn imo

File: config/game_options.txt [hunk 1]
```diff
@@ -392,7 +392,7 @@ JOIN_WITH_MUTANT_RACE
 ## Uncommenting races will allow them to be choosen at roundstart while join_with_muntant_race is on. You'll need at least one.
 
 ## You probably want humans on your space station, but technically speaking you can turn them off without any ill effect
-ROUNDSTART_RACES human
+#ROUNDSTART_RACES human
```
Comment (DoctoreMad) [inline config/game_options.txt]: Make sure to merge instead of closing

**Comment by sawrge** [comment]
I have a bunch of games to offer for this and future bounties, I'll just make a forum thread about it here:
https://tgstation13.org/phpBB/viewtopic.php?f=5&t=13922&p=356151#p356150

**Comment by Robustin** [comment]
I wouldn't even call this a "redo" to RND. We're removing RND and adding a totally different station-wide economy. 

If I weren't on the verge of burning out again **I'd create a GBP-like mechanic for every department that allows them to assist the "central" RND system. This would also incorporate the randomization that @Rinoka was talking about and would function like miniature station goals, but for a department. One round the Medbay Lathe has a research project involving a difficult virus combo, a set of 6 specific disabilities in a DNA injector, and 100 units of a rare chemical like Strange Reagent... next round it might require infecting a majority of the crew with a virus, creating a monkey human with a certain appearance, etc...**

Completing these projects would result in the immediate completion of a technology for that department, perhaps the tech "reward" would also be randomized to prevent it just being a cog in the machine of rushing certain "meta" technologies. Or perhaps certain "endgame" tech could only be unlocked by completing some of these projects. 

**Likewise I'd suggest incorporating the GBP-esque feature of having the department able to furnish their own resources. Like a universal "budget" currency that can be used as a gap-filler for missing resources. So if you can't get ______ mineral from the QM you can simply use 100 units from your budget to produce the item at your lathe, perhaps the budget could also be used to "rush" tech from that department too. I already fleshed out this idea for engineering but its with enough work it could be done for any department. Perhaps that budget could also be transferable to other departments, useful as a negotiating instrument to secure cooperation (Science will wire you 1000 GBPs for a canister of Tritium, Engineering).**

I think it would be a huge boon to gameplay, crew interaction, but also avoiding what are some of the inevitable pitfalls of making every dept. competing for resources that might not even be coming out of cargo. Perhaps redeemed resources should be split evenly or within tightly controlled ranges and we just balance costs so that no department has a resource glut or famine, and like the budget it could be transferred to meet a specific need (i.e. Warops). Maybe the QM could deviate from equal distribution but any department that sees their "fair share" diminished (its a zero sum thing after all) would require consent from either the Captain or the "diminished" department head. 

Anyway these are all ideas that will take longer than a 2 week freeze to accomplish, but I absolutely think the framework should be ready to incorporate them because a straightforward "RND decides who to bless with tech, QM decides who to bless with resources, remaining departments reduced to sitting in their office and begging on radio" is an almost criminal under-utilization of what this system could offer.

**Comment by praisenarsie** [comment]
kor, just add the economy subsystem already

**Comment by KorPhaeron** [comment]
I guess we know what phase 3 is now

**Comment by Sarcalogo** [comment]
Does GBP stand for good boy points..?

**Comment by davethwave** [comment]
By the way, if we are going to give cargo a super lathe then we should probably create some custom science tech just for that purpose

**Comment by davethwave** [comment]
And I mean the cargo tech side of science, not the miner side

**Comment by ike709** [comment]
@Sarcalogo Yes.

**Comment by kevinz000** [comment]
robustin go

**Comment by Repukan** [comment]
The frog croaks sadly

**Comment by Bawhoppen** [comment]
It probably won't be a very good system but anything is better than current R&D so

**Comment by obscolene** [comment]
@KorPhaeron specifications on the lathe sprites would be appreciated

**Comment by KorPhaeron** [comment]
Something similar to the current autolathe or protolathe but coloured for each department

Just need to recolour sprites for the most part

**Comment by calibraptor** [comment]
I'm actually excited for this.

When do we get to test merge decentralized RnD?

**Comment by davethwave** [comment]
It's a set of individual pr's so you can't just test merge it. It's done in steps, not one huge pr.

**Comment by davethwave** [comment]
> Maybe something like that, plus the PR mentions adding custom-made bots - maybe add some upgraded/custom settings for bots to make integrated circuitry bots? Or a way to make a spawner for material delivery bots? It'd be interesting to see how that went, even if it had a high metal cost limiting how many could be made easily.
I think they are something that cargo could probably order instead

**Comment by Armhulen** [comment]
@KorPhaeron can i add a hud to the game (like plasmafire and slimecore)

**Comment by KorPhaeron** [comment]
Sprites are exempt yeah

**Comment by davethwave** [comment]
how long do we have to wait before we can merge integrated circuits and techwebs!

**Comment by KorPhaeron** [comment]
We testmerged both today so probably within the next week or so I hope

**Comment by davethwave** [comment]
nice, too bad I wasn't there too see them both testmerged though

**Comment by JJRcop** [comment]
What do you feel about balance changes that are number tweaking, like this? #32690 (I personally disapprove of the PR, but I think @Cyberboss closing it due to this freeze is not in line with the spirit of the freeze.)

**Comment by optimumtact** [comment]
it's only two weeks, suck it up buttercup

**Comment by davethwave** [comment]
Well both integrated circuits and techwebs are pretty much done besides some minor bugs that people are finding now and then so it should soon hopefully that they will be merged finally.

**Comment by AnturK** [comment]
Spring now

**Comment by Qustinnus** [comment]
feelsgoodman

**Pull request closed without merging**